### ISSUE TITLE
env files are not loaded/saved

### ISSUE THREAD

**Issue by tringler-atos** [rangav/thunder-client-support#1424]
**Describe the bug**
If import an env file the content is not loaded. If I link an env file and click on save the content is not wrote to that file.
If I export an env file it is working.

**To Reproduce**
Export an env file and try to re-import it.

**Expected behavior**
The env file content is loaded/saved.

**Platform:** 
 - OS: Win11
 - vscode version: 1.84.2
 - extension version: 2.16.0

**Are you using the free version/paid version/trial:**
paid

**Comment by rangav** [comment]
Hi @tringler-atos

I just now exported the environment from TC and imported the env file. It's working for me.

Are you referring to `.env` files or TC environment files?

**Comment by tringler-atos** [comment]
I'm referring to .env files - These you can create beside of the global and local TC env files.

**Comment by rangav** [comment]
- I import .env file named .env.test and its working for me
- I linked a .env file and saved, and used a variable it's working

The extension will not write back to .env file when you link it. its a read only operation that is by design.

**Comment by tringler-atos** [comment]
Import worked now. 
I removed the env from TC, added a new variable on the json and imported back to TV. After reloading TC I'm getting `Sidebar Load Error: Cannot read properties of undefined (reading 'replace')`

If I remove the env file from environments folder it's back working. As I imported the env file the variable I manually edited is visible, so it seems to be deserialized correctly.

Any idea?
